x-pack/plugins/security_solution/public/resolver/view/process_event_dot.tsx

```diff
@@ -405,22 +450,27 @@ const ProcessEventDotComponents = React.memo(
         return [];
       }
       // If we have entries to show, map them into options to display in the selectable list
-      return Object.entries(relatedStats).map((statsEntry) => {
-        const displayName = getDisplayName(statsEntry[0]);
-        return {
-          prefix: <EuiI18nNumber value={statsEntry[1] || 0} />,
-          optionTitle: `${displayName}`,
-          action: () => {
-            dispatch({
-              type: 'userSelectedRelatedEventCategory',
-              payload: {
-                subject: event,
-                category: statsEntry[0],
-              },
-            });
-          },
-        };
-      });
+      const statsList = [];
+      for (const stat in relatedStats) {
```

Comment:
:grey_question: If we're going for performance here, `for const stat of Object.keys(relatedStats)` would let you dodge the call to `hasOwnPoperty` below with the same result, I think.